pkg/querier/blocks_store_queryable_test.go

```diff
@@ -489,8 +489,8 @@ func TestBlocksStoreQuerier_Select(t *testing.T) {
 				},
 			},
 			limits:       &blocksStoreLimitsMock{maxChunksPerQuery: 3},
```

Comment:
General Comment: If we don't use the `BlocksStoreLimits` is it worth keeping it as part of the test struct?